src/tr/eldermanga/src/eu/kanade/tachiyomi/extension/tr/eldermanga/ElderManga.kt

```diff
@@ -0,0 +1,237 @@
+                val element = elements[i]
+
+                // Manga başlığı
+                val titleElement = element.select("h2")
+                if (titleElement.isEmpty()) continue
+                val title = titleElement[0].text()
+
+                // Manga URL'si
+                val linkElements = element.select("a")
+                if (linkElements.isEmpty()) continue
+                val mangaUrl = linkElements[0].attr("href")
+                if (!mangaUrl.contains("/manga/")) continue
+
+                // Manga kapak resmi
+                val imgElements = element.select("div.card-image img")
+                val thumbnail = if (imgElements.isNotEmpty()) imgElements[0].attr("src") else ""
+
+                // En son bölüm bilgisi
+                val chapterElements = element.select("div.flex.pt-0.flex-col.gap-2\\.5 > a")
+                val hasNewChapter = chapterElements.isNotEmpty()
+
+                if (hasNewChapter) {
+                    val manga = SManga.create().apply {
+                        this.title = title
+                        setUrlWithoutDomain(mangaUrl)
+                        thumbnail_url = when {
+                            thumbnail.startsWith("https://") || thumbnail.startsWith("http://") -> thumbnail
+                            thumbnail.startsWith("//") -> "https:$thumbnail"
+                            thumbnail.isEmpty() -> null
+                            thumbnail.startsWith("/") -> baseUrl + thumbnail
+                            else -> "https://cdn1.eldermanga.com/$thumbnail"
+                        }
+                    }
+                    mangas.add(manga)
+                }
+            } catch (e: Exception) {
+                // Hata durumunda pas geç
+            }
+        }
+
+        return MangasPage(mangas, false)
+    }
+
+    override fun fetchSearchManga(
+        page: Int,
+        query: String,
+        filters: FilterList,
+    ): Observable<MangasPage> {
+        if (query.startsWith(URL_SEARCH_PREFIX)) {
+            val url = "$baseUrl/manga/${query.substringAfter(URL_SEARCH_PREFIX)}"
+            return client.newCall(GET(url, headers)).asObservableSuccess().map { response ->
+                val document = response.asJsoup()
+                when {
+                    isMangaPage(document) -> MangasPage(listOf(mangaDetailsParse(document)), false)
+                    else -> MangasPage(emptyList(), false)
+                }
+            }
+        }
+        return super.fetchSearchManga(page, query, filters)
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val url = "$CDN_URL/series/search/navbar".toHttpUrl().newBuilder()
+            .addQueryParameter("search", query)
+            .build()
+        return GET(url, headers)
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        val dto = json.decodeFromString<List<SearchDto>>(response.body.string())
```

Comment:
You can use utils functions too

```suggestion
        val dto = response.parseAs<List<SearchDto>>()
```